mmsplice/vcf_dataloader.py

```diff
@@ -1,312 +1,104 @@
-import pickle
 import warnings
-import functools
-
-import gffutils
-from pyfaidx import Fasta
+from itertools import islice
+import pandas as pd
+import pyranges
+from pybedtools import Interval
```

Comment:
I would not add pybedtools as dependencies if possible. We have an 'Interval' class implementation which is compatible with pybedtools.Interval

Reply:
Until we implement interval and variant in kipoiseq #30, I prefer  `pybedtools.Interval` in the current release. Because classes such as `MultiSampleVCF`, `VariantSeqExtractor` from kipoiseq require  pybedtools.Interval.